### DISCUSSION TITLE
3D Printed Enclosure for P6 64x32 LED MATRIX

### DISCUSSION THREAD

**Discussion by jeffwalker15** [riffnshred/nhl-led-scoreboard#301] [General]
Thank you so much for creating the NHL scoreboard and for keeping everything up to date.  I'm a beginner in all of this, and the instructions and support for this project are great.  Our family has really enjoyed having the NHL scoreboard this past year.  

We recently finished our basement with a home theater room, and we plan to hang the NHL scoreboard on the wall with other Calgary Flames and movie memorabilia.  We've used a P6 64x32 LED Matrix that we purchased through adafruit. I was wondering if anyone has leads on where I can purchase a 3D printed enclosure for a p6 LED Matrix.  I've done some searches on some of the websites mentioned on here but have come up empty so far.  

If there's nothing out there, I'd definitely appreciate any recommendations for how to create an enclosure or how to best hang the scoreboard on our wall in a way that looks finished/nice.

**Comment by pjockey** [comment]
Look on Thingiverse.com.

**Reply by riffnshred** [reply]
No P6 anywhere sadly :P

**Comment by riffnshred** [comment]
Thanks for the good word man, I really appreciate it.

As for a case for that size I never found any. In your case I think the best would be some kind of bracket that your can screw on the panel so you can hang it. Im sure you could find something that would work at the hardware store. 

Do you have a Skadis Peg board by any chance? I designed and printed some hooks for that. 

![20210811_174005](https://user-images.githubusercontent.com/16941896/129108945-f4d78d9a-ba25-4c0d-a41d-bd8dc7cd1f23.jpg)
![20210811_173408](https://user-images.githubusercontent.com/16941896/129108957-2181e4d6-bb11-4625-ac56-e306c497ed54.jpg)
